plugin/src/main/java/io/jenkins/plugins/casc/ConfigurationAsCode.java

```diff
@@ -712,11 +712,24 @@ private static void invokeWith(Mapping entries, ConfiguratorOperation function)
         }
 
         if (!entries.isEmpty()) {
-            final Map.Entry<String, CNode> next = entries.entrySet().iterator().next();
-            throw new ConfiguratorException(format("No configurator for root element <%s>", next.getKey()));
+            List<String> unknownKeys = new ArrayList<>();
+            entries.entrySet().iterator().forEachRemaining(next -> {
+                String key = next.getKey();
+                if (ignoreAliasEntry(key)) {
+                    unknownKeys.add(key);
+                }
+            });
+
+            if (!unknownKeys.isEmpty()) {
+                throw new ConfiguratorException(format("No configurator for the following root elements %s", String.join(", ", unknownKeys)));
+            }
         }
     }
 
+    static boolean ignoreAliasEntry(String key) {
```

Comment:
Naming is hard 😓 But at least I wanted to give the condition a name.

Reply:
This condition seems the wrong way round to me from the name, isNotAliasEntry?